Service/DataService.php

```diff
@@ -237,16 +238,17 @@ public function propagateDataToComputedField(FormInterface $form, array& $object
         return $this->propagateDataToComputedFieldRecursive($form, $objectArray, $contentType, $type, $ouuid, $migration, $objectArray, '');
     }
 
-    private function propagateDataToComputedFieldRecursive(FormInterface $form, array& $objectArray, ContentType $contentType, $type, $ouuid, $migration, &$parent, $path)
+    private function propagateDataToComputedFieldRecursive(Form $form, array& $objectArray, ContentType $contentType, $type, $ouuid, $migration, &$parent, $path)
```

Comment:
Why does the Form class is better than the Interface? does the function use non-interface function?

Reply:
Yes ->getIterator on line 334
(bullet 3 in my commit message https://github.com/ems-project/EMSCoreBundle/pull/176/commits/22071a276125ec3b8e6fcdb94c78e3fc60cc20bb )

Reply:
I found a better solution ! see last commit

Reply:
Ok to me